artifacts/pom.xml

```diff
@@ -102,20 +112,26 @@
 		<demosdk-client.location>/usr/share/nginx/html/artifactory/libs-release-local/demosdk/default/demosdk</demosdk-client.location>
 		<demosdk-client.fileName>demosdk-client-jar-with-dependencies.jar</demosdk-client.fileName>
 		<!-- mock-sdk -->
-		<mock-sdk.version>1.2.0.2</mock-sdk.version>
+		<mock-sdk.version>1.3.0-SNAPSHOT</mock-sdk.version>
 		<mock-sdk.location>/usr/share/nginx/html/artifactory/libs-release-local/mock-sdk/1.1.5</mock-sdk.location>
 		<mock-sdk.fileName>mock-sdk.jar</mock-sdk.fileName>
-		<!-- mock-sdk-jpeg-extractor -->
-		<mock-sdk-jpeg-extractor.version>1.2.0.1-SNAPSHOT</mock-sdk-jpeg-extractor.version>
-		<mock-sdk-jpeg-extractor.location>/usr/share/nginx/html/artifactory/libs-release-local/jpeg-mock-sdk/1.1.5</mock-sdk-jpeg-extractor.location>
-		<mock-sdk-jpeg-extractor-lib.location>/usr/share/nginx/html/artifactory/libs-release-local/jpeg-sdk/jpeg-sdk-lib</mock-sdk-jpeg-extractor-lib.location>
-		<mock-sdk-jpeg-extractor.fileName>mock-sdk-jpeg-extractor-jar-with-dependencies.jar</mock-sdk-jpeg-extractor.fileName>
+		<mock-sdk-dep.fileName>mock-sdk-jar-with-dependencies.jar</mock-sdk-dep.fileName>
+		<!-- image-compressor-->
+		<image-compressor.version>0.1.0-SNAPSHOT</image-compressor.version>
+		<image-compressor.location>/usr/share/nginx/html/artifactory/libs-release-local/compressor/image-compressor</image-compressor.location>
+		<image-compressor.fileName>image-compressor-jar-with-dependencies.jar</image-compressor.fileName>
+
+		<!-- redis-cache -->
+		<redis-cache.version>1.0-SNAPSHOT</redis-cache.version>
+		<redis-cache.location>/usr/share/nginx/html/artifactory/libs-release-local/cache</redis-cache.location>
+		<redis-cache.fileName>redis-cache-provider.jar</redis-cache.fileName>
+
 		<!-- hazelcast cast -->
 		<hazelcast.version>3.12.12</hazelcast.version>
 		<hazelcast.location>/usr/share/nginx/html/artifactory/libs-release-local/cache</hazelcast.location>
 		<hazelcast.fileName>cache-provider.jar</hazelcast.fileName>
 		<!-- authentication-childauthfilter-impl -->
-		<authentication-childauthfilter-impl.version>1.2.0.1</authentication-childauthfilter-impl.version>
+		<authentication-childauthfilter-impl.version>1.2.0.2</authentication-childauthfilter-impl.version>
```

Comment:
is it required authentication-childauthfilter-impl because we have removed from mosip-ref-impl

Reply:
This is not required. We will be removing the same as part of below JIRA:
https://mosip.atlassian.net/browse/MOSIP-36413